models/path.py

```diff
@@ -42,10 +42,14 @@ def choose_vlans(self, controller, old_path_dict: dict = None):
         If any of the links next tag isn't available, it'll release
         all vlans of the path that have been allocated, all or nothing.
         """
+        if not self:
+            return
         old_path_dict = old_path_dict if old_path_dict else {}
+        topology_napp = controller.napps[("kytos", "topology")]
```

Comment:
Let's handle the `KeyError` that can happen if topology is loaded out of order or gets unloaded. None of these are normally expected, but if it ever happens, let's let this not to crash, and then at the call sites here they at least log the error, that that the `topology` dependency isn't available. See what I mean? 

In the future we'll have standardize exceptions for this kind of stuff, but since we haven't got there yet, and we're making this flexibility here to facilitate to land this change, let's try to have at least part of that exc handling covered too.